src/MinimalDelegation.sol

```diff
@@ -117,7 +117,8 @@ contract MinimalDelegation is
         returns (uint256 validationData)
     {
         _payEntryPoint(missingAccountFunds);
-        (bytes32 keyHash, bytes memory signature) = abi.decode(userOp.signature, (bytes32, bytes));
+        (bytes32 keyHash, bytes memory signature, bytes memory hookData) =
+            abi.decode(userOp.signature, (bytes32, bytes, bytes));
```

Comment:
ewwww

Reply:
hmm yeah thats not cute, also in the 4337 path there will be hookData in the execute path too